apps/payments/next/config.ts

```diff
@@ -0,0 +1,478 @@
+      default: 'fxa-payments-broker',
+      format: 'String',
+      env: 'SENTRY_SERVER_NAME',
+    },
+    clientName: {
+      doc: 'Name used by sentry to identify the client.',
+      default: 'fxa-payments-client',
+      format: 'String',
+      env: 'SENTRY_CLIENT_NAME',
+    },
+  },
+  servers: {
+    auth: {
+      url: {
+        default: 'http://localhost:9000',
+        doc: 'The url of the fxa-auth-server instance',
+        env: 'AUTH_SERVER_URL',
+        format: 'url',
+      },
+    },
+    content: {
+      url: {
+        default: 'http://localhost:3030',
+        doc: 'The url of the corresponding fxa-content-server instance',
+        env: 'CONTENT_SERVER_URL',
+        format: 'url',
+      },
+    },
+    oauth: {
+      url: {
+        default: 'http://localhost:9000',
+        doc: 'The url of the corresponding fxa-oauth-server instance',
+        env: 'OAUTH_SERVER_URL',
+        format: 'url',
+      },
+      clientId: {
+        default: '59cceb6f8c32317c',
+        doc: 'The Payments frontend OAuth client id.',
+        env: 'OAUTH_SUBSCRIPTIONS_CLIENT_ID',
+        format: 'String',
+      },
+    },
+    profile: {
+      url: {
+        default: 'http://localhost:1111',
+        doc: 'The url of the corresponding fxa-profile-server instance',
+        env: 'PROFILE_SERVER_URL',
+        format: 'url',
+      },
+    },
+    profileImages: {
+      url: {
+        default: 'http://localhost:1112',
+        doc: 'The url of the Firefox Account Profile Image Server',
+        env: 'FXA_PROFILE_IMAGES_URL',
+        format: 'url',
+      },
+    },
+    accountsCdn: {
+      url: {
+        default: 'https://accounts-static.cdn.mozilla.net',
+        doc: 'The URL of the Firefox Accounts static content CDN',
+        env: 'FXA_STATIC_CDN_URL',
+        format: 'url',
+      },
+    },
+  },
+  staticResources: {
+    directory: {
+      default: 'build',
+      doc: 'Directory where static resources are located',
+      env: 'STATIC_DIRECTORY',
+      format: String,
+    },
+    maxAge: {
+      default: '10 minutes',
+      doc: 'Cache max age for static assets, in ms',
+      env: 'STATIC_MAX_AGE',
+      format: 'duration',
+    },
+    url: {
+      default: 'http://localhost:3031',
+      doc: 'The origin of the static resources',
+      env: 'STATIC_RESOURCE_URL',
+      format: 'url',
+    },
+  },
+  statsd: {
+    enabled: {
+      doc: 'Enable StatsD',
+      format: Boolean,
+      default: false,
+      env: 'STATSD_ENABLE',
+    },
+    sampleRate: {
+      doc: 'Sampling rate for StatsD',
+      format: Number,
+      default: 1,
+      env: 'STATSD_SAMPLE_RATE',
+    },
+    maxBufferSize: {
+      doc: 'StatsD message buffer size in number of characters',
+      format: Number,
+      default: 500,
+      env: 'STATSD_BUFFER_SIZE',
+    },
+    host: {
+      doc: 'StatsD host to report to',
+      format: String,
+      default: 'localhost',
+      env: 'DD_AGENT_HOST',
```

Comment:
Not sure if this is legacy, but it looks like it isn't the standard `STATSD_` prefix like the previous ones.

Reply:
This is currently what is in our other packages so I'm going to keep all of the `STATSD_` stuff for now. It might get removed during 3.0 build. Same for all related comments.